ui/app/hooks/useSwapsEthToken.js

```diff
@@ -0,0 +1,42 @@
+import { useSelector } from 'react-redux'
+import { getSelectedAccount } from '../selectors'
+import { ETH_SWAPS_TOKEN_OBJECT } from '../helpers/constants/swaps'
+import { getValueFromWeiHex, hexToDecimal } from '../helpers/utils/conversions.util'
+
+/**
+ * @typedef {Object} SwapsEthToken
+ * @property {string} decimals - a secondary currency string formatted for display
+ * @property {string} symbol - the symbol for ETH, namely "ETH"
+ * @property {string} name - the name of the eth current, "Ether"
```

Comment:
Typo, and slight improvement
```suggestion
 * @property {string} name - the name of the Ether currency, "Ether"
```